_posts/2015-06-26-ux-part-2.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: post
+title: The OME UX Team. Part 2 - The OME-UX Development Workflow
+categories: user-experience
+---
+*By Gus Ferguson*
+
+In this post we will run through a real workflow that has been done over the last few releases of OMERO.
+
+The diagram below outlines our day-to-day pragmatic approach to developing.
+
+<img src="/images/workflow.png" alt="A diagram of the workflow of OME User Centred Design" style="width: 600px;"/>
+
+*Diagram of workflow of User Centred Design.*
+
+##Requirements Gathering
+
+Most of the UX team work starts with requirements in some form or another. This can range from a formal request through our forums or mailing lists to informal requests from users at meetings or even people sidling up to Jason in the pub and telling him they need OMERO to do the next big thing.
+
+The requirement can often be a single use-case from a bench scientist, ranging from a simple task through to a large, complex workflow. This can be entirely new or a request for enhancement (RFE) of an existing workflow or piece of functionality. Commonly, the requirement is specific to a relatively small number of users, and they are often very busy people and may be located in many different parts of the world.
+
+This means it can be a challenge spending enough time with a wide enough range of users to ensure the requirements are not too narrow and generalise to be applicable to the widest range of users and use cases as possible. However we generally find that once people are on board they are very generous with their time and expertise.
+
+##Client Group Meetings
+
+Our workflow revolves around our Client Group meetings where members of the team present requirements, talk through issues and present the results of any scoping for feedback. And this is where we usually relive the classic tussle between conceptual and functional designs.
+
+The traditional approach is that the conceptual design stage it’s all blue-sky, open-ended, ideal-world thinking. The idea is not to close off any avenues at an early stage and instead try and think about what the system should really do and how it should really work. All fluffy, cuddly and cute. 
```

Comment:
should be an "at" between "traditional approach is that" and "the conceptual design…" ?